examples/build.gradle

```diff
@@ -42,6 +42,9 @@ clean {
 }
 
 task testExamples {
+    if (project.gradle.startParameter.excludedTaskNames.contains('test')) {
+        return
+    }
```

Comment:
We don't need this since we are already added this to the test task.

Reply:
removed